$OpenBSD$

./configure[7906]: syntax error: `(' unexpected
./configure[8107]: ${!name}": bad substitution
./configure[8107]: declare: not found

Index: configure
--- configure.orig
+++ configure
@@ -7902,21 +7902,8 @@ ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
-if test -n "$enable_s3"; then
-  declare -a aws_version_tokens=($(printf '#include <aws/core/VersionConfig.h>\nAWS_SDK_VERSION_STRING' | $CPP $CPPFLAGS - | grep -v '^#.*' | sed 's/"//g' | tr '.' ' '))
 
-cat >>confdefs.h <<_ACEOF
-#define AWS_VERSION_MAJOR ${aws_version_tokens[0]}
-_ACEOF
 
-
-cat >>confdefs.h <<_ACEOF
-#define AWS_VERSION_MINOR ${aws_version_tokens[1]}
-_ACEOF
-
-fi
-
-
 # Whether to use the Boehm garbage collector.
 # Check whether --enable-gc was given.
 if test "${enable_gc+set}" = set; then :
@@ -8113,11 +8100,6 @@ fi
 # Expand all variables in config.status.
 test "$prefix" = NONE && prefix=$ac_default_prefix
 test "$exec_prefix" = NONE && exec_prefix='${prefix}'
-for name in $ac_subst_vars; do
-    declare $name="$(eval echo "${!name}")"
-    declare $name="$(eval echo "${!name}")"
-    declare $name="$(eval echo "${!name}")"
-done
 
 rm -f Makefile.config
 
